Float.Core/Float.Core.csproj

```diff
@@ -38,6 +38,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json" Version="[12.0.1,]" />
+    <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.2" />
```

Comment:
@arctouch-alejandrovarela Would it make sense to adjust the version here to be a range? (i.e. `[8.2.2,9)`) so we don't have to update this library every time a new Mvvm version comes out?